### PR TITLE
Simplify kotlin example. Remove parent, properties

### PR DESCRIPTION
**Pull request by Tradunsky**

### CODE REVIEW

**Comment by Tradunsky** [comment]
[#178](https://github.com/takari/polyglot-maven/issues/178)

File: kotlin/pom.kts [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
-        "project.build.sourceEncoding" assign "UTF-8"
-        "junit.version" += 4.12
-        "kotlin.version" assign "1.1.61"
-    }
+    val kotlinVersion = "1.1.61"
 
     dependencies {
-        compile("org.jetbrains.kotlin:kotlin-stdlib:" + it["kotlin.version"])
+        compile("org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion")
         compile(it.groupId + ":polyglot-common:" + it.version)
```
Comment (mosabua) [inline kotlin/pom.kts]: why is this needed as dependency for a normal project? It should not be needed

File: kotlin/pom.kts [hunk 2]
```diff
@@ -1,29 +1,25 @@
...
 
         test(
                 "junit:junit:$junitVersion" exclusions "org.hamcrest:hamcrest-core",
-                "org.jetbrains.kotlin:kotlin-test-junit:${it["kotlin.version"]}"
+                "org.jetbrains.kotlin:kotlin-test-junit:$kotlinVersion"
         )
         provided(
                 "io.takari.polyglot:polyglot-kotlin:0.3.3-SNAPSHOT",
-                "org.apache.maven.plugin-tools:maven-plugin-annotations:LATEST"
+                "org.apache.maven.plugin-tools:maven-plugin-annotations:3.3"
```
Comment (mosabua) [inline kotlin/pom.kts]: Why is this here?

Comment (Tradunsky) [reply kotlin/pom.kts]: could be removed. Was added as example of LATEST

Comment (Tradunsky) [reply kotlin/pom.kts]: Actually, there is no dependency that actually required here, all of them made up

File: kotlin/pom.kts [hunk 3]
```diff
@@ -1,29 +1,21 @@
...
-        "kotlin.version" assign "1.1.61"
-    }
+    val kotlinVersion = "1.1.61"
 
     dependencies {
-        compile("org.jetbrains.kotlin:kotlin-stdlib:" + it["kotlin.version"])
-        compile(it.groupId + ":polyglot-common:" + it.version)
-                .exclusions("org.slf4j:jul-logger")
+        compile("org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion")
+        compile("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.0.0")
```
Comment (Tradunsky) [inline kotlin/pom.kts:13]: this is just to provide an example of exclusions in pipeline style, no real need in this dependency

**Comment by Tradunsky** [comment]
@mosabua Please let me know if there open question left
